src/components/Button/Button.js

```diff
@@ -1,12 +1,12 @@
 import React from 'react';
-import './Button.css';
+import './Button.css'
 
-const Button = () => {
-  return (
-    <div className='btn-dataview'>
+const Button = ()=>{
+  return(
+    <div>
```

Comment:
These changes break tests because the className is what the test is looking for. Also, the removal of the semi-colon's kick back ESLint errors.